code/rewriter.go

```diff
@@ -21,13 +21,15 @@ import (
 	"go/token"
 	"os"
 	"path/filepath"
+	"runtime/debug"
 	"strings"
 )
 
 const (
-	packagePath  = "github.com/pingcap/failpoint"
-	packageName  = "failpoint"
-	evalFunction = "Eval"
+	packagePath         = "github.com/pingcap/failpoint"
+	packageName         = "failpoint"
+	evalFunction        = "Eval"
+	failpointNameGetter = "__failpointName"
```

Comment:
We should choose a good name for this, any suggestion?
@kennytm @amyangfei @winkyao

Reply:
How about `__pkg__`